src/storage/media.js

```diff
@@ -0,0 +1,33 @@
+const config = require('../config');
+
+const mediaTypes = {
+  EVENT_COVER: 1,
+  PROFILE_PICTURE: 2,
+};
+
+const getMediaTypeConfig = (mediaType) => {
+  let mediaTypeConfig;
+  switch (mediaType) {
+    case mediaTypes.EVENT_COVER:
+      mediaTypeConfig = {
+        type: mediaTypes.EVENT_COVER,
+        maxFileSize: config.file.MAX_EVENT_COVER_FILE_SIZE,
+        uploadPath: config.file.EVENT_COVER_UPLOAD_PATH,
+        fileTag: 'image',
+      };
+      break;
```

Comment:
Return instead of breaking so we can throw an error if the input doesn't get pattern matched.